lib/cc/engine/analyzers/violation.rb

```diff
@@ -32,12 +39,24 @@ def report_name
           "#{current_sexp.file}-#{current_sexp.line}"
         end
 
+        def mass # ABH Reverse engineer the actual mass from flay score.
+          if issue.identical?
+            flay_score / (occurrences.length**2)
```

Comment:
:lipstick: I'm surprised rubocop didn't complain about lack of space around the `**` operator.